staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/helpers.go

```diff
@@ -439,6 +439,9 @@ func (UnstructuredObjectConverter) Convert(in, out, context interface{}) error {
 func (UnstructuredObjectConverter) ConvertToVersion(in runtime.Object, target runtime.GroupVersioner) (runtime.Object, error) {
 	if kind := in.GetObjectKind().GroupVersionKind(); !kind.Empty() {
 		gvk, ok := target.KindForGroupVersionKinds([]schema.GroupVersionKind{kind})
+		// if gvk.GroupVersion().Version == runtime.APIVersionInternal {
+		// 	return in, nil
```

Comment:
@smarterclayton I wanted to treat the original version as the internal version of a custom resource. Otherwise we need special handling for `Unstructured` at https://github.com/kubernetes/kubernetes/pull/56606/files#diff-15bdfc59ec78ff09b509d4a347560a1bR244 and https://github.com/kubernetes/kubernetes/pull/56606/files#diff-15bdfc59ec78ff09b509d4a347560a1bR346.

Reply:
Looks much better without these lines. But we should delete instead of comment out :)